.phpcs.xml

```diff
@@ -4,7 +4,7 @@
 
     <file>gebrueder-weiss-woocommerce.php</file>
     <file>includes/</file>
-    <file>tests</file>
+    <file>tests/</file>
```

Comment:
is this a typie?

Reply:
no - i want tests/ directory to be added. like the includes/ directory